### PR TITLE
Latentchecker

### PR DESCRIPTION
**Pull request by smd111**
@Byrth
this addon checks the weaponskill points that have been accumulated by you ws trial and mythic weapons

sence byrth wrote the main functions and i wrote the the unequip and return to invitory functions i would like to get his/her approval before this is added to the addon list

### CODE REVIEW

**Comment by Byrth** [comment]
Looks good to me, particularly if it works! I know the script originally worked because I unlocked Blade: Ku with it.

**Comment by z16** [comment]
You're missing an `addons.xml` entry for this addon. Also, @Byrth are you ok with him attributing it to you? It'll make you the person to contact for this addon's bugs and support.

As for the code, there are a few things that could be more stable, I'll review it later today.

**Comment by smd111** [comment]
added addons.xml entry

File: addons/addons.xml [hunk 1]
```diff
@@ -482,3 +482,10 @@
     <support>http://mibbit.com/#windower@irc.ffochat.com</support>
   </addon>
 </addons>
+<addon>
+    <name>latentchecker</name>
+    <author>byrth,smd111</author>
+    <description>Checks weapon skill points of weapon skill tiral weapons and mythic weapons</description>
```
Comment (z16) [inline addons/addons.xml]: `s/tiral/trial`

**Comment by smd111** [comment]
sorry dislexia is a bear

**Comment by z16** [comment]
The addon is also missing the [mandatory copyright disclaimer](https://github.com/Windower/Lua#licensing). Add that somewhere in your code files (all files, if you have more than one).

**Comment by smd111** [comment]
added

File: addons/latentchecker/latentchecker.lua [hunk 2]
```diff
@@ -0,0 +1,95 @@
+--[[Copyright © <year>, <your name>
```
Comment (z16) [inline addons/latentchecker/latentchecker.lua]: You need to fill in the year and your name here.

File: addons/latentchecker/latentchecker.lua [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of <addon name> nor the
```
Comment (z16) [inline addons/latentchecker/latentchecker.lua:12]: And here.

File: addons/latentchecker/latentchecker.lua [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL <your name> BE LIABLE FOR ANY
```
Comment (z16) [inline addons/latentchecker/latentchecker.lua]: And here.

**Comment by smd111** [comment]
i think thats all of them

**Pull request merged**